pnpm-lock.yaml

```diff
@@ -10,6 +10,12 @@ dependencies:
     version: 3.9.0-rc.1(@types/react@18.2.55)(graphql@16.8.1)(react-dom@18.2.0)(react@18.2.0)
   class-variance-authority:
     specifier: ^0.7.0
+<<<<<<< HEAD
```

Comment:
why we have  updated pnm lock file. please double check it. if it is just reordering then its fine. :)

Reply:
it is just reordering